src/main/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/HaplotypeCallerArgumentCollection.java

```diff
@@ -312,6 +319,21 @@ boolean isFlowBasedCallingMode() {
         return flowMode != FlowMode.NONE;
     }
 
+    // Default constructor
+    public HaplotypeCallerArgumentCollection() {
+        super();
+    }
+
+    // Private constructor for use in copyWithNewPloidy
+    // Allows you to make new instances of hcArgs with synced fields except for genotyper ploidy
+    private HaplotypeCallerArgumentCollection(int ploidy) {
+        this.standardArgs.genotypeArgs.samplePloidy = ploidy;
+    }
+
+    // Copy method used to create new hcArgs with same fields except input ploidy model
+    public HaplotypeCallerArgumentCollection copyWithNewPloidy(int ploidy) {
```

Comment:
I don't see how this copies the fields. . . it looks like it would set all fields except the given ploidy to their defaults.  This could use a unit test.

Reply:
I could have sworn there was a post online suggesting this paradigm for copying with one value changed, but.as it goes I can't find it again. It seems to have worked when I tried it, but maybe it's not the best way to do it. Do you have any other suggestions for trying to do this in a safer way?